### PR TITLE
Fix Layout/LineLength - Bundle 1

### PR DESCRIPTION
**Pull request by macanudo527**
#### First Bundle of Layout/LineLength Rubocop Offense Fixes

- Helps #6055 
- This is a bundle of fixes that were originally attached to #10714 

This makes code easier to read on smaller screens by reducing the line length. These fixes are bundled in groups of 60 to make it easier to review.

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Comment by macanudo527** [comment]
All of the rubocop errors will probably be resolved once #10714 goes through.

**Comment by jibees** [comment]
Sill in Draft, and some checks failed, moving to `In Dev` status.

**Comment by sigmundpetersen** [comment]
Hey @macanudo527 this is from our [Zenhub](https://www.zenhub.com/) board.
You need to be invited to our board, I'll let @lin-d-hop answer about the process

**Comment by macanudo527** [comment]
I cleaned this one up and rebased it. It should be good to go. Thanks!

File: app/services/products_renderer.rb
Comment (dacook) [inline app/services/products_renderer.rb:72]: `±` This format is harder to read. 
And I just noticed that it is unnecessary. This was wrapped at 80 chars, but our rubocop_styleguide is set 100.

**Review by dacook — CHANGES_REQUESTED** [review]
So many good updates, thanks for this!

It's unclear how much (if any) was automated with `rubocop --autocorrect`. 
I think it would be easier to review if you were to commit that first (we can assume rubocop auto corrections are "safe"), then apply manual corrections as a separate commit.

Also I noticed some corrections were wrapping at the unnecessarily low width of 80 chars. We allow 100 chars (
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/.rubocop_styleguide.yml#L78-L80), which is necessary when you see some of the extremely long variable and class names we have!

I've made a couple of comments marked with `±` where I think changes should be made.

Thanks.

File: app/controllers/admin/enterprises_controller.rb
Comment (macanudo527) [reply app/controllers/admin/enterprises_controller.rb:329]: At this point, I'm not sure how this came to be. I'm going to chart it up to a mistake on my part and I need to make smaller commits/PRs so that it is easier to keep track of everything. Lesson learned. I'm going to revert it.

File: app/controllers/admin/product_import_controller.rb
Comment (macanudo527) [reply app/controllers/admin/product_import_controller.rb:47]: Thanks!

File: app/mailers/subscription_mailer.rb
Comment (macanudo527) [reply app/mailers/subscription_mailer.rb:42]: I have yet to learn the wonders of `[].join(" ")` I'll keep it in mind for next time, though.

File: app/models/spree/app_configuration.rb
Comment (macanudo527) [reply app/models/spree/app_configuration.rb:79]: Should I change this to `{ mini: "48x48>", small: "100x100>", product: "240x240>", large:"600x600\>"}.to_json`, then?

File: app/models/spree/order.rb
Comment (macanudo527) [reply app/models/spree/order.rb:150]: That really makes it very compact.

File: app/models/spree/app_configuration.rb
Comment (dacook) [reply app/models/spree/app_configuration.rb:79]: Sorry I was ambiguous there. 
The generated string looks different, so I'm not sure 100% sure about changing it. I think it's not worth changing at this point.

File: app/models/spree/order.rb
Comment (dacook) [reply app/models/spree/order.rb:150]: Wow, it does but I think it broke something!

Many spec failures, I'd start with the quickest one to execute:
```
rspec spec/queries/complete_visible_orders_spec.rb
```

Perhaps we still need three lines?

**Review by dacook — CHANGES_REQUESTED** [review]
Thanks for the updates, but unfortunately something broke

**Comment by macanudo527** [comment]
Should I regenerate a `rubocop_todo.yml` to eliminate the `Style/HashSyntax` errors?

**Comment by dacook** [comment]
Yes, good idea. I think they are appearing now (and not always) because the linter only runs on changed files. I guess these files haven't been changed in a long time.

Hopefully we can fix those in a future round :)

**Comment by macanudo527** [comment]
I'm not getting the `Style/HashSyntax` errors locally. I'll have to investigate as to what is causing them.

**Comment by macanudo527** [comment]
@dacook OK, this looks clean and ready. Thanks for your help!

**Review by dacook — APPROVED** [review]
Ah of course, yes the linter and tests run on a merge with the master branch (which I actually find unintuitive and a bit confusing).

All good 👍

File: app/controllers/admin/enterprises_controller.rb
Comment (rioug) [inline app/controllers/admin/enterprises_controller.rb:126]: I think this change is a bit confusing, I'd prefer something like this : 
```suggestion 
          render(
              json: @collection,
              each_serializer: Api::Admin::ForOrderCycle::EnterpriseSerializer,
              order_cycle: @order_cycle,
              spree_current_user: spree_current_user
          )
```

**Comment by macanudo527** [comment]
@rioug should I merge this with master? I'm not sure of the protocol. Do I need two approvals and can I merge it myself?

**Comment by mkllnk** [comment]
Our process is described in the wiki:

* https://github.com/openfoodfoundation/openfoodnetwork/wiki/The-process-of-review%2C-test%2C-merge-and-deploy

Summary:

* PR submitted
* Two approvals in Code Review (by core devs)
* Manual testing on a staging server (by core tester)
* Merge into master (by core tester or core dev)

**Review by mkllnk — APPROVED** [review]
Thank you!

Very nice size for a pull request.

**Review by mkllnk — APPROVED** [review]
I think that we can skip the manual testing in this case. There's always manual release testing before any code goes to production as well.

**Comment by macanudo527** [comment]
> I think that we can skip the manual testing in this case. There's always manual release testing before any code goes to production as well.

Thanks for being patient with me and helping me learn! I really appreciate it.

**Pull request merged**
